src/main/scala/com/fulcrumgenomics/sv/tools/SvPileup.scala

```diff
@@ -21,12 +21,45 @@ import scala.collection.mutable
   """
     |Collates a pileup of putative structural variant supporting reads.
     |
+    |## Outputs
```

Comment:
this whole usage was really tough to write, it could use a thorough "not me" read through